src/js/shield_defs.js

```diff
@@ -450,8 +450,8 @@ export function loadShields() {
     padding: {
       left: 4,
       right: 4,
-      top: 5,
-      bottom: 5,
+      top: 6,
+      bottom: 4,
```

Comment:
```suggestion
      bottom: 5,
```

There's enough room to squash this vertically by one pixel without affecting icon selection. I think this looks a little better for 3+ digits. Before/after:

![image](https://github.com/ZeLonewolf/openstreetmap-americana/assets/1732117/7e7ee192-93f0-4d9b-98b8-a52c956e852a)

![image](https://github.com/ZeLonewolf/openstreetmap-americana/assets/1732117/91cbca69-d785-4ea3-a0fa-ac80740506ad)

Reply:
LGTM